taichi/backends/cc/cc_program.cpp

```diff
@@ -1,18 +1,54 @@
-#include "taichi/common/core.h"
-#include "taichi/program/kernel.h"
-#include "taichi/program/program.h"
-#include "taichi/system/dynamic_loader.h"
-#include "taichi/util/action_recorder.h"
-#include "struct_cc.h"
-#include "cc_program.h"
-#include "cc_runtime.h"
-#include "cc_kernel.h"
-#include "cc_layout.h"
-#include "cc_utils.h"
-#include "context.h"
+#include "taichi/backends/cc/cc_program.h"
+
+using namespace taichi::lang::cccp;
```

Comment:
It's better to use `namespace cccp {`?

Reply:
The APIs here are for ProgramImpl, which does not belongs to cccp scope. Moreover, this is consistent with other unified backends I believe.